libgstd/libgstd.c

```diff
@@ -243,3 +244,65 @@ gstd_free (GstD * gstd)
   g_object_unref (gstd->session);
   g_free (gstd);
 }
+
+GstdReturnCode
+gstd_create (GstD * gstd, const gchar * uri, const gchar * name,
+    const gchar * description)
+{
+  GstdReturnCode ret = GSTD_EOK;
+  GstdObject *node = NULL;
+
+  g_return_val_if_fail (NULL != gstd, GSTD_NULL_ARGUMENT);
+  g_return_val_if_fail (NULL != gstd->session, GSTD_NULL_ARGUMENT);
+  g_return_val_if_fail (NULL != uri, GSTD_NULL_ARGUMENT);
+  g_return_val_if_fail (NULL != name, GSTD_NULL_ARGUMENT);
```

Comment:
`description` pointer guard

Reply:
This was intentional because not all `create` need `description`. For example an EOS event:
`
gstd_create (manager, "/pipelines/p1/event", "eos", NULL);
`

Reply:
ok, understood